benchmarks/distributed/rpc/ps/bash_experiment_scripts/ddp_nccl_allreduce.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# configuration ids
+bconfig_id=3
+dconfig_id="DummyData"
+mconfig_id="DummyModel"
+tconfig_id="DdpNcclTrainer"
+pconfig_id="None"
+
+
+# moves to directory and runs the benchmark with the configurations selected
+cd "$(dirname $(dirname "$0"))"
+source ./bash_experiment_scripts/helper_functions.sh
```

Comment:
may want to note here or in the README that you need slurm to run this script

Reply:
ill add it to the script.